apps/examples/testcase/le_tc/kernel/tc_wqueue.c

```diff
@@ -0,0 +1,165 @@
+* COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+* INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+* BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+* OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+* AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+* LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+* ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*
+**************************************************************************/
+
+/**************************************************************************
+* Included Files
+**************************************************************************/
+#include <tinyara/config.h>
+#include <tinyara/wqueue.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/types.h>
+#include <ctype.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+
+/**************************************************************************
+* Private Definitions
+**************************************************************************/
+
+
+#if defined(CONFIG_SCHED_WORKQUEUE) || defined(CONFIG_LIB_USRWORK)
+
+
+
+/**************************************************************************
+* Private Types
+**************************************************************************/
+
+/**************************************************************************
+* Private Function Prototypes
+**************************************************************************/
+
+/**************************************************************************
+* Global Variables
+**************************************************************************/
+
+/**************************************************************************
+* Private Variables
+**************************************************************************/
+
+static systime_t start_time;
+
+/**************************************************************************
+* Private Functions
+**************************************************************************/
+static void wq_test1(void *arg)
+{
+	printf("worker thread 1 \n");
+}
+
+static void wq_test2(void *arg)
+{
+	printf("worker thread 2 \n");
+}
+
+static void wq_test3(void *arg)
+{
+	printf("worker thread 3 \n");
+}
+/**************************************************************************
+* Public Functions
+**************************************************************************/
+#if defined(CONFIG_SCHED_HPWORK) || defined(CONFIG_SCHED_LPWORK)
+static void tc_wqueue_cancel(void)
+{
+	int result;
+	struct work_s *test_work1;
+	struct work_s *test_work2;
+	struct work_s *test_work3;
+
+	test_work1 = (struct work_s *)malloc(sizeof(struct work_s));
+	test_work2 = (struct work_s *)malloc(sizeof(struct work_s));
+	test_work3 = (struct work_s *)malloc(sizeof(struct work_s));
+
+	start_time = clock_systimer();
```

Comment:
for clock_systimer, please add the proper header file

Reply:
will add it